src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -149,7 +151,9 @@ function MdAutocomplete () {
       floatingLabel:    '@?mdFloatingLabel',
       autoselect:       '=?mdAutoselect',
       menuClass:        '@?mdMenuClass',
-      inputId:          '@?mdInputId'
+      inputId:          '@?mdInputId',
+      noBlurOnEscape:   '=?mdNoBlurOnEscape',
```

Comment:
If possible, I like to invert the logic here so that the the option is `md-blur-on-escape` with a default of `true` instead of `md-no-blur-on-escape` with a default of `false`. Same for the `md-no-clear-on-escape` below.

Is it easy/possible to update these here?